big_tests/tests/domain_removal_SUITE.erl

```diff
@@ -101,7 +101,7 @@ group_to_modules(offline_removal) ->
 group_to_modules(vcard_removal) ->
     [{mod_vcard, [{backend, rdbms}]}];
 group_to_modules(last_removal) ->
-    [{mod_last, [{backend, rdbms}]}].
+    [{mod_last, #{iqdisc => one_queue, backend => rdbms}}].
```

Comment:
This is a small issue, but I'd go with `[{mod_last, config_parser_helper:mod_config(mod_last, #{backend => rdbms})}].` here, and `default_mod_config(mod_last)` in places where no specific options are set. This way it's more visible which options are really needed for a test, and the defaults can be changed easily in the future.

Reply:
It's maybe a bit more verbose, but I'd say not that bad